src/isomorphic/children/__tests__/ReactChildren-test.js

```diff
@@ -227,7 +223,7 @@ describe('ReactChildren', function() {
 
     var instance = (
       <div>{
-        [frag({
+        [ReactFragment.create({
```

Comment:
Later in the code, there's the test:

```js
   it('should warn if a fragment is accessed', function() {
     spyOn(console, 'error');
     var child = React.createElement('span');
     var frag = ReactChildren.map([child, child], function(c) {
       return c;
     });
     ...
```

Which means there's shadowing. Since frag was just a stub method used in only two places, I figured it'd be better to just get rid of it rather than renaming things to avoid a conflict.